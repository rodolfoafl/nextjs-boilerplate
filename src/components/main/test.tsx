import Main from '@/components/main'

import { render, screen } from '@testing-library/react'

describe('<Main />', () => {
  it('should render the heading', () => {
    const { container } = render(<Main />)

    expect(
      screen.getByRole('heading', { name: /main component/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
