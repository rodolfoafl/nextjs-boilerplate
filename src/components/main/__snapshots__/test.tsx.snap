// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render the heading 1`] = `
<main
  class="bg-color-[#06092b] text-white w-full h-full p-12 text-center flex flex-col items-center justify-center"
>
  <h1>
    Main Component
  </h1>
</main>
`;
